pom.xml

```diff
@@ -110,6 +122,24 @@
       <artifactId>vertx-unit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
the dependency on slf4j is from the smtp mock server that is used for tests, without this it will not display any messages, we could replace this by slf4j-simple instead of log4j (this is a run-time dependency of the test)